libnavigation-router/src/main/java/com/mapbox/navigation/route/offboard/router/MapboxDirectionsUtils.kt

```diff
@@ -31,7 +34,9 @@ internal fun MapboxDirections.Builder.routeOptions(
         }
     }
 
-    alternatives(options.alternatives() ?: false)
```

Comment:
Removing these we're not opinionated anymore (unless user explicitly calls `applyRecommendedOptions`). These options are `null` by default and we're not falling back (`?:`) i.e. these options only change when they're provided beforehand. Is that what we want? 🤔 

The same thing applies to the other removals 👇

Reply:
This https://github.com/mapbox/mapbox-navigation-android/pull/4320#discussion_r626612065 still applies. What do you think?